daemons/gptp/common/ieee1588port.cpp

```diff
@@ -107,7 +107,7 @@ IEEE1588Port::IEEE1588Port(IEEE1588PortInit_t *portInit)
 		if (initialLogSyncInterval == LOG2_INTERVAL_INVALID)
 			initialLogSyncInterval = -5;     // 31.25 ms
 		if (initialLogPdelayReqInterval == LOG2_INTERVAL_INVALID)
-			initialLogPdelayReqInterval = -3;  // 125 ms
```

Comment:
Was this perhaps meant to be the operLogSyncInterval instead?  Having the sync interval at 1 second actually seems to cause problems too.